src/main/java/com/synopsys/integration/alert/provider/polaris/descriptor/PolarisGlobalDescriptorActionApi.java

```diff
@@ -73,4 +80,14 @@ public void testConfig(final TestConfigModel testConfig) throws IntegrationExcep
             throw new AlertException(ioException);
         }
     }
+
+    @Override
+    public FieldModel saveConfig(final FieldModel fieldModel) {
+        final Optional<AccessTokenPolarisHttpClient> polarisHttpClient = polarisProperties.createPolarisHttpClientSafely(logger);
+        final Optional<String> nextRunTime = taskManager.getNextRunTime(BlackDuckAccumulator.TASK_NAME);
+        if (polarisHttpClient.isPresent() && nextRunTime.isEmpty()) {
+            taskManager.scheduleCronTask(PolarisProjectSyncTask.DEFAULT_CRON_EXPRESSION, PolarisProjectSyncTask.TASK_NAME);
+        }
+        return super.saveConfig(fieldModel);
```

Comment:
Similar to what I said previously, this seems to be something that should be abstracted out now. Anytime we have a scheduled provider, we will need to perform this funcitonality. So perhaps a parent class or a utility class to perform this would help a bunch.